README.md

```
@@ -106,7 +106,42 @@ You can plug in your own toast layouts, define duration policies, and limit how
+    ToastDuration.Indefinite -> null
+}
+
+KoffeeBar(
+    modifier = Modifier.padding(innerPadding),
+    config = myConfig,
+) {
+    NavHost()
+}
+```
+
+The old method is not deprecated yet. See it below
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)